File: FCUBEAPI/FleetTrans/Models/TripOutstandingRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class TripOutstandingRptListModel
    {
        public List<TripOutstandingRptModel> TripOutstandingRptlist { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: FCUBEAPP/src/app/models/tripoutstandingrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Tripoutstandingrptmodel } from "./tripoutstandingrptmodel";

export class Tripoutstandingrptlistmodel {
    tripOutstandingRptlist: Tripoutstandingrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
